ttnn/cpp/pybind11/operations/__init__.hpp

```diff
@@ -69,8 +72,11 @@ void py_module(py::module& module) {
     auto m_conv2d = module.def_submodule("conv2d", "conv2d operation");
     conv2d::py_module(m_conv2d);
 
-    auto m_maxpool2d = module.def_submodule("maxpool2d", "maxpool 2d operation");
-    maxpool2d::py_module(m_maxpool2d);
+    auto m_maxpool = module.def_submodule("maxpool", "maxpool  operation");
+    maxpool::py_module(m_maxpool);
+
+    auto m_avgpool = module.def_submodule("avgpool", "avgpool  operation");
+    avgpool::py_module(m_avgpool);
```

Comment:
I think this should go to conv? submodule.

Reply:
please do not create modules for maxpool and avgpool. in you want to group, group things into the conv module

Reply:
I think it would better to create a pool submodule for max pool and average poll, as the pooling ops are significantly different from convolutions.

Reply:
pool ops should not be in conv, they should be in their own module (`pool`)